### PR TITLE
EU-Parliament Constituencies for Irleand

### PR DESCRIPTION
**Pull request by sguenther85**
added EU-Parliament Constituencies for Irleand and also for the EU election 2024 in Irleand. 
These are the first constituencies which are not the same like for the lower house of the country. 
So the whole thing is also open for discussion around the right type for the paths. I used "eu" right now, because the type described it the best for which usecase they are. But let me know if you have better suggestions.

### CODE REVIEW

**Comment by sguenther85** [comment]
@azuser @jloutsenhizer Can anybody have a first look. Would be great if we can clarify this year how we want to deal with extra eu election constituencies of a country.
We have after that three other countries to add for the eu election.

File: identifiers/country-ie/README.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# OCD Division Identifier
+
```
Comment (HKSenior) [inline identifiers/country-ie/README.md:2]: Can you add a `Types` section listing out the various types used in this folder. For eg: https://github.com/opencivicdata/ocd-division-ids/blob/master/scripts/country-ca/README.md?plain=1#L23

File: identifiers/country-ie/README.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+This folder list the following administrative divisions of Ireland:
+
+* Lower House Constituencies [ref](https://en.wikipedia.org/wiki/D%C3%A1il_constituencies). Used type: **ed**.
```
Comment (HKSenior) [inline identifiers/country-ie/README.md:5]:
```suggestion
* Lower House Constituencies [ref](https://en.wikipedia.org/wiki/D%C3%A1il_constituencies).
```

You can remove this (here and below) once the `Type` section is added above.

File: identifiers/country-ie.csv [hunk 3]
```diff
@@ -39,3 +39,6 @@ ocd-division/country:ie/ed:tipperary,Tipperary
 ocd-division/country:ie/ed:waterford,Waterford
 ocd-division/country:ie/ed:wexford,Wexford
 ocd-division/country:ie/ed:wicklow,Wicklow
+ocd-division/country:ie/eu:dublin,Dublin
```
Comment (HKSenior) [inline identifiers/country-ie.csv:42]: I'd recommend using `eup` (for EU Parliament) instead of `eu` for the type.

**Review by HKSenior — CHANGES_REQUESTED** [review]

**Comment by jloutsenhizer** [comment]
+1 to @HKSenior's suggestions, otherwise this PR looks good to me.

File: identifiers/country-ie/README.md [hunk 4]
```diff
@@ -0,0 +1,6 @@
+# OCD Division Identifier
```
[the remark this answers is not in the document]
Comment (sguenther85) [reply identifiers/country-ie/README.md]: Followed your suggestion. Send an update. Please have a look.

File: identifiers/country-ie/README.md (hunk 1 above)
Comment (sguenther85) [reply identifiers/country-ie/README.md:2]: Followed your suggestion. Send an update. Please have a look.

File: identifiers/country-ie/README.md (hunk 2 above)
Comment (sguenther85) [reply identifiers/country-ie/README.md]: Followed your suggestion. Send an update. Please have a look.

File: identifiers/country-ie.csv (hunk 3 above)
Comment (sguenther85) [reply identifiers/country-ie.csv]: Followed your suggestion. Send an update. Please have a look.

**Review by HKSenior — APPROVED** [review]

**Comment by sguenther85** [comment]
Hey together, please wait with the merge. 
There is a discussion on DPA/Google side where we have to add the constituencies for the eup election.
This way, or in the country-eu.csv with all other constituencies which belongs the the eup election.

**Comment by sguenther85** [comment]
Google/DPA: Sorry, but we discussed this internally and the EUP constituencies should be in the eu-countries file. Basically all of the districts for the EU Parliament (those that have jurisdiction of the EU) should be in one file.

So i will close this pull request and create a new one with all constituencies which will be used for the eup election.

**Pull request closed without merging**